docs/source/index.js

```diff
@@ -82,6 +82,8 @@ docs['docs/jobs.pug'] = {
 docs['docs/change-streams.md'] = { title: 'MongoDB Change Streams in NodeJS with Mongoose', markdown: true };
 docs['docs/lodash.md'] = { title: 'Using Mongoose with Lodash', markdown: true };
 docs['docs/incompatible_packages.md'] = { title: 'Known Incompatible npm Packages', markdown: true };
+docs['docs/check-version.md'] = { title: 'How to Check Your Mongoose Version', markdown: true };
+docs['docs/version-support.md'] = { title: 'Version Support, markdown: true };
```

Comment:
```suggestion
docs['docs/version-support.md'] = { title: 'Version Support', markdown: true };
```

my previous suggestion missed the closing quote